test/Sentry.Tests/SerializationTests.Serialization_TransactionAndSpanData.verified.txt

```diff
@@ -0,0 +1,44 @@
+﻿{
+  EventId: {},
+  SpanId: {},
+  ParentSpanId: {},
+  Name: name,
+  Platform: csharp,
+  StartTimestamp: DateTimeOffset_1,
+  Operation: operation,
+  Description: ,
+  IsSampled: false,
+  SampleRate: 0.0,
+  Request: {},
+  Contexts: {
+    trace: {
+      SpanId: {},
+      ParentSpanId: {},
+      Operation: operation,
+      Description: ,
+      IsSampled: false,
+      Extra: {
```

Comment:
@aritchie this is suppsed to be `Data` now right (not `Extra`)?

Reply:
This appears to be the result of letting verify do the serialization.  I did this in order to let it scrub the dates.  The built-in ToJsonString functions are generating the data outputs.  I'll see what I can do with the test.

Reply:
I've moved the tests to just json path walk to ensure the tree is serialized properly